target/scripts/startup/fixes-stack.sh

```diff
@@ -34,17 +34,17 @@ function _fix_var_amavis_permissions
   [[ ! -e ${AMAVIS_STATE_DIR} ]] && return 0
 
   _notify 'inf' 'Fixing Amavis permissions'
-  chown -hR amavis:amavis "${AMAVIS_STATE_DIR}" || _shutdown 'Failed to fix Amavis permissions'
+  chown -hR amavis:amavis "${AMAVIS_STATE_DIR}" || _notify 'err' 'Failed to fix Amavis permissions'
```

Comment:
Same here..

Reply:
In this case, I don't think this should be a fatal error because peration of DMS is "fine" even without Amavis.

Reply:
Sure? Do you know, how postfix behaves when amavis is enabled, but not working due to wrong permissions?

Reply:
I don't know. Feel free to make it a fatality as well:) My gut tells me that jut logging the error is fine though:)

Reply:
@polarathene What's your take on this?

Reply:
Wow, quick response :D I'm fine with you changing it to a fatal error, really. When I think about it, having a fatal error may be the best option after all - fail fast is a good approach!

Reply:
> @polarathene What's your take on this?

No opinion as I'm not too familiar of the impact :)

If it's important to do this for functionality to be correct, then do so. We can always revert later with a patch release if it's problematic.

Alternatively, if just raising an error in the logs is sufficient... that should still surface in any bug reports?

Reply:
> No opinion as I'm not too familiar of the impact :)

Same as Georg and me 😆

I'll revert to `_shutdown` for another reason. Even when postfix doesn't care if amavis is not working, I see a kind of security risk.
Spamassassin & ClamAV will not scan emails. Therefore potential dangerous content can be delivered to user mailboxes.

Reply:
Let us try it like this. As long as it has no unwanted effects, it should be good to stay as a fatal error.

Reply:
> > No opinion as I'm not too familiar of the impact :)
> 
> Same as Georg and me 😆
> 
> I'll revert to `_shutdown` for another reason. Even when postfix doesn't care if amavis is not working, I see a kind of security risk. Spamassassin & ClamAV will not scan emails. Therefore potential dangerous content can be delivered to user mailboxes.

Let us try it like this. As long as it has no unwanted effects, it should be good to stay as a fatal error.